src/chitterApi.js

```diff
@@ -0,0 +1,103 @@
+class chitterApi {
+  constructor() {
+    this.apiUrl = "https://chitter-backend-api-v2.herokuapp.com/peeps";
+    this.apiCreateUserUrl =
+      "https://chitter-backend-api-v2.herokuapp.com/users";
+    this.apiLoginUserUrl =
+      "https://chitter-backend-api-v2.herokuapp.com/sessions";
+    this.apiCreatePeppUrl =
```

Comment:
nit - typo `Pepp` > `Peep`
You can use a spellChecker extension on your IDE to check variable names.

Reply:
Absolutely yes